client/src/components/App.jsx

```diff
@@ -0,0 +1,27 @@
+import Header from './Header.jsx';
+import Ratings from './Ratings.jsx';
+import Reviews from './Reviews.jsx';
+
+class App extends React.Component {
+
```

Comment:
The tabs/spacing on this component looks off. Did you run your entire app through ESLint?